### PR TITLE
DOC-3459, DOC-3465 Update learner topics about certs and course pacing

### PR DESCRIPTION
**Pull request by catong**
## [DOC-3465](https://openedx.atlassian.net/browse/DOC-3465), [DOC-3459](https://openedx.atlassian.net/browse/DOC-3459)

Refreshed the SFD topics about types of certs and receiving certificates. Modified the linked "About Self-paced courses" topic to provide info about both self-paced and instructor-paced courses.

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [x] Doc team review: @edx/doc 
FYI: @sstack22, @marcotuts 

### HTML Drafts

- [x] http://draft-sfd-certs-pacing.readthedocs.io/en/latest/SFD_certificates.html
- [x] http://draft-sfd-certs-pacing.readthedocs.io/en/latest/SFD_self_paced.html

### Testing

- [x] Ran ./run_tests.sh - known errors due to new anchors

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [x] Squash commits

### CODE REVIEW

**Comment by catong** [comment]
Note to reviewers - tests will fail until content containing the new anchors is published on RTD.

File: en_us/students/source/SFD_certificates.rst [hunk 1]
```diff
@@ -87,167 +79,137 @@ Verified certificates include the following information.
...
-
-If a course offers on-demand certificates, as soon as you have qualified for a
-certificate, you see the following message at the top of the course
-**Progress** page.
-
-::
-
-  Congratulations, you've qualified for a certificate!
+======================
+On Demand Certificates
```
Comment (grantgoodmanedX) [inline en_us/students/source/SFD_certificates.rst]: "On-demand" with a hyphen, to be consistent with other uses?

**Review by grantgoodmanedX — APPROVED** [review]
Looks good to me, with one trivial comment. +1

**Comment by catong** [comment]
Merging https://github.com/edx/edx-documentation/pull/1442

**Pull request merged**